nodejs/week1/avg.js

```diff
@@ -0,0 +1,18 @@
+const arguments = process.argv;
```

Comment:
arguments could be a reserved javascript variable so change it to another name
https://www.w3schools.com/js/js_reserved.asp